core/common/src/Clock.kt

```diff
@@ -10,19 +10,18 @@ import kotlin.time.*
 /**
  * A source of [Instant] values.
  *
- * @see Clock.System for the instance that queries the operating system.
+ * See [Clock.System][Clock.System] for the clock instance that queries the operating system.
```

Comment:
How is `[Clock.System][Clock.System]` different from `[Clock.System]`?

Reply:
Can be rendered as just `System` by dokka. So it is to ensure the exact text of the link label.